src/com/xilinx/rapidwright/design/DesignTools.java

```diff
@@ -3926,8 +3938,45 @@ public static void prohibitPartialHalfSlices(Design design) {
                         bottomUsed = true;
                     }
                 }
+                if (c.getBEL().isFF()) {
```

Comment:
It seems like this new code doesn't fit the ethos of the original method name (prohibiting half slices) -- does it make sense to move it out to another standalone method that could be called without prohibiting anything?

At the very least, let's update the javadoc.

Reply:
Refactored the name and updated Javadoc.